src/libs/actions/ReimbursementAccount/index.js

```diff
@@ -5,6 +5,7 @@ import setupWithdrawalAccount from './setupWithdrawalAccount';
 import fetchFreePlanVerifiedBankAccount from './fetchFreePlanVerifiedBankAccount';
 import resetFreePlanBankAccount from './resetFreePlanBankAccount';
 import deleteFromBankAccountList from './deleteFromBankAccountList';
+import updatePersonalInformationForBankAccount from './updatePersonalInformationForBankAccount';
```

Comment:
We've been having the refactored commands in [BankAccount.js](https://github.com/Expensify/App/blob/9496c4fa71dc386404b3600295f52f1de4a3e30f/src/libs/actions/BankAccounts.js) instead of in separate JS files. I don't really see a need for each command to have an individual file. We can then have getVBBADataForOnyx() back in BankAccount.js as well.